src/components/MessageThumbnail.tsx

```diff
@@ -0,0 +1,37 @@
+import {FC} from "react";
+import Image from "next/image";
+
+interface userStatus{
```

Comment:
nitpick: move the helper interfaces below the primary interface - they are arguments supporting a thesis